vite.config.ts

```diff
@@ -27,6 +27,8 @@ const viteconfig = ({ mode }) => {
     defaultCensusSize = 5000
   }
 
+  let stripePublicKey = process.env.STRIPE_PUBLIC_KEY || ''
```

Comment:
No need to to this here (do it directly in the JSON.stringify part), plus there's no need for a `let`. A `const` would be better.